features/openhab-addons-external/src/main/resources/conf/knx.cfg

```diff
@@ -60,3 +60,7 @@
 # Seconds to wait for an orderly shutdown of the auto refresher's 
 # ScheduledExecutorService. (optional, defaults to 5)
 #scheduledExecutorServiceShutdownTimeoutString=
+
+# Use NAT (Network Address Translation)
+#  (optional; defaults to false)
+#useNAT=
```

Comment:
Considering that's not done for any of the other boolean entries, and that
it's only done for 1 knx entry at all, you're right, that seemed
unnecessary.

In fact, some of the entries are missing an equals sign.

But I did it anyway.

HTH...